### DISCUSSION TITLE
Matrix-Chat Channel beside or with bridge to Slack?

### DISCUSSION THREAD

**Discussion by cf-ka** [sinatra/sinatra#1738] [General]
Hey together,
it's only a short question to the Sinatra-Community.
What do you think about a Matrix channel beside the Slack-Chat?
First of all, I know it's more difficult to maintain two chats or more communitys-places and that this  it will be annoying that this question come up because the next one want a discord channel. :)
But I want to ask it anyway, because Matrix have the possibility to set up a bridge to the slack-chat and maybe for some other people interests in Sinatra is this a other, easy or more flexible way to enter/contact the community. 

Let me know what you all think about it.
I hope this is the right place to ask this. :)

Greetings cf-ka

**Comment by epergo** [comment]
In my opinion having too many communication channels for an open source project is counter-productive, users of the library with issues won't know where is the best place to post their problem and maybe the person that knows the solution is only reading other platform. I know we can create the bridge but...it is another thing to _maintain_

If the Sinatra community gets reactivated enough and there is a clear need of supporting Matrix, sure! but for now I don't think is worth it (IMO). We could start by fixing the invitation form of the Slack community because is broken and I cannot join :sweat_smile:
